ldap/src/main/java/org/springframework/security/ldap/userdetails/LdapUserDetailsManager.java

```diff
@@ -403,6 +403,15 @@ public void setSecurityContextHolderStrategy(SecurityContextHolderStrategy secur
 		this.securityContextHolderStrategy = securityContextHolderStrategy;
 	}
 
+	/**
+	 * Sets the role prefix used when converting authorities. The default value is "ROLE_"
+	 * @param rolePrefix role prefix
+	 * @since 6.3
+	 */
+	public void setRolePrefix(String rolePrefix) {
```

Comment:
Will you please check for null in this method? You can use `Assert#notNull` for this.

Reply:
done